python/dgl/backend/pytorch/sparse_emb.py

```diff
@@ -0,0 +1,160 @@
+import torch as th
+from .tensor import attach_grad, is_recording
+from .util import get_shared_mem_array, create_shared_mem_array
+from datetime import timedelta
+
+from ... import ndarray as nd
+
+_store = None
+
+class NodeEmbedding: # NodeEmbedding
+    '''Sparse node embeddings for graph.
```

Comment:
I wonder if there is such a thing called sparse embeddings. The embeddings are actually dense. its gradient updates are sparse. I think we should avoid using "sparse embeddings" to confuse people.